### ISSUE TITLE
 Undefined array key "logits"

### ISSUE THREAD

**Issue by mankowitz** [CodeWithKyrian/transformers-php#67]
### System Info

Windows 11
Laravel 10

### PHP Version

8.1.28

### Environment/Platform

- [ ] Command-line application
- [X] Web application
- [ ] Serverless
- [ ] Other (please specify)

### Description

I created an onnx version of a pytorch model on hf called `mankowitz/bert-large-uncased_med-ner` using the colab jupyter notebook. I want to use it to identify medication names in a string.

When I execute the name recognition, I get an error 
```
Undefined array key "logits"
```

### Reproduction

In PHP, I do the following:
```
use function Codewithkyrian\Transformers\Pipelines\pipeline;
$ner = pipeline('token-classification', 'mankowitz/bert-large-uncased_med-ner');
$string = "synthroid, ViT D3, celexa, abilify, claritin, keppra 500mg bid, tegretol 100mg bid, atarax. ";
$output = $ner($string, aggregationStrategy: 'max');
```

I get the following error:

```
Undefined array key "logits"

  at vendor\codewithkyrian\transformers\src\Models\Output\TokenClassifierOutput.php:21
     17▕     }
     18▕
     19▕     public static function fromOutput(array $array): self
     20▕     {
  ➜  21▕         return new self($array['logits']);
     22▕     }
     23▕ }
```
